docs/README.md

```diff
@@ -63,20 +63,21 @@ Open [localhost:3004](http://localhost:3004) to view the docs.
 
 All markdown files are part of [this repository](https://github.com/grafana/grafana). However, all images are added to the [website repository](https://github.com/grafana/website). Therefore, the process of adding images is not as straightforward. These are the steps:
 
-1. Ensure you create a feature branch within the [website repository](https://github.com/grafana/website) to make the change. This branch needs to be based on the `old-docs` branch. 
-2. Ensure the image(s) are compressed and optimised e.g. Using [tinypng](https://tinypng.com/)
-2. Add the image(s) to the `/static/img/docs` directory
-3. Then, make a commit that adds the image(s)
-4. The Pull Request you create needs to target where you branched off, the branch `old-docs`
+1. Ensure you create a feature branch within the [website repository](https://github.com/grafana/website) to make the change. This branch needs to be based on the `old-docs` branch.
+1. Ensure the image(s) are compressed and optimised e.g. Using [tinypng](https://tinypng.com/).
+1. Add the image(s) to the `/static/img/docs` directory.
+1. Then, make a commit that adds the image(s).
+1. The Pull Request you create needs to target where you branched off, the branch `old-docs`.
```

Comment:
Since this is a set of ordered instructions that need to be checked as a whole, (as opposed to a list of full-stop statements) I would argue they do not want the period at the end of each sentence. Instead, one change we could make to make this a bit more clear would be to lowercase the first word of each sentence to give the reader an impression they follow one from the other. 

I'm a bit biased since I wrote this 😆I'll leave the decision in your hands on whenever you want to change this or not.

Reply:
From the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/writing-step-by-step-instructions):

> Use a period after each step.

If you feel strongly about this, we should add it to the documentation style guide, to clarify when the period is needed, and when it's not.

Reply:
Absolutely not, please disregard my comment and I have made a mental note to use it from now on. 

Although it is not my preference, I would rather adhere to the guidelines as-is as opposed to try and bend them at this point. This is why having guidelines is great -- thank you for all this hard work!